streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedTable.java

```diff
@@ -35,6 +38,8 @@
      * @param adder         the instance of {@link Reducer} for addition
      * @param subtractor    the instance of {@link Reducer} for subtraction
      * @param name          the name of the resulted {@link KTable}
+     * @return a {@link KTable} with the same key and value types as this {@link KGroupedTable}
```

Comment:
"Only paranoid survives" :) I am not native speakers, but I think no comma before "containing" is fine.